lib/google/gax/grpc.rb

```diff
@@ -72,22 +74,23 @@ module Grpc
       # @return A gRPC client stub.
       def create_stub(service_path,
                       port,
-                      chan_creds: nil,
                       channel: nil,
+                      chan_creds: nil,
                       updater_proc: nil,
                       scopes: nil)
         address = "#{service_path}:#{port}"
-        if channel.nil?
-          chan_creds = GRPC::Core::ChannelCredentials.new if chan_creds.nil?
+        if channel
+          yield(address, nil, channel_override: channel)
+        elsif chan_creds
```

Comment:
I think the old way was right, and this is wrong. It's totally OK to specify both a `chan_creds` (credentials on the channel, like SSL) and an updater proc (credentials on the call, like OAuth).